z3-sys/src/lib.rs

```diff
@@ -3098,6 +3098,9 @@ extern "C" {
     /// Retrieve from `s` the unit sequence positioned at position `index`.
     pub fn Z3_mk_seq_at(c: Z3_context, s: Z3_ast, index: Z3_ast) -> Z3_ast;
 
+    /// Retrieve from s the element positioned at position index. The function is under-specified if the index is out of bounds.
```

Comment:
How does this differ from `Z3_mk_seq_at`?

Also, some backticks should be used like in the other docstrings for `s` and `index`.

Reply:
Fixed the docstrings. Also added a sentence about the difference to the high level binding docstring.